Src/Eiffel/interface/new_graphical/dialogs/es_breakpoint_dialog.e

```diff
@@ -725,6 +725,17 @@ feature {NONE} -- When hits
 						af.set_item_size (avb, af.width - 1, af.height - horizontal_scrollbar_height - 1)
 					end(l_fixed, vb, ?,?,?,?)
 				)
+			l_scroll.dpi_changed_actions.extend (agent (asc: EV_SCROLLABLE_AREA; a_dpi,ax,ay,aw,ah: INTEGER)
+					do
+						asc.set_item_size (asc.item.minimum_width.max (asc.width - vertical_scrollbar_width), asc.item.minimum_height.max (asc.height - horizontal_scrollbar_height))
```

Comment:
I am wondering, if this code could not be moved to EV_SCROLLABLE_AREA under a new feature ?
maybe "update_size"  or similar. 
This way, you could reuse it for resize_actions on l_scroll.  This avoids code duplication, it would make the executable a (tiny) bit smaller, ...

Reply:
I see, but I don't know what to do with the extra argument for `dpi`, at the moment it's not used.